pkg/scheduler/scheduler_test.go

```diff
@@ -1403,6 +1403,237 @@ func TestSchedule(t *testing.T) {
 				"eng-beta": {"eng-beta/older_new"},
 			},
 		},
+		"minimal preemptions when target queue is exhausted": {
```

Comment:
remove either the first or second test case, as the specifics are covered in preemption_test.go